### ISSUE TITLE
> ok... This another little things.. but not from last phase...

### ISSUE THREAD

**Issue by slonyarakmw** [puzzlehunt/gsmgio-5btc-puzzle#7]
> ok... This another little things.. but not from last phase...
> ![2021-05-17_19-58-48](https://user-images.githubusercontent.com/40121870/118528240-c70e5400-b74a-11eb-91d0-6b83a4818e59.png)
> ![2021-05-17_19-53-23](https://user-images.githubusercontent.com/40121870/118528245-c83f8100-b74a-11eb-91cf-effe2a980e0b.png)
> 
> i don't know what do with it. I trying to decode but may be it another thing.....

open a new issue for this

_Originally posted by @edoardocoen in https://github.com/puzzlehunt/gsmgio-5btc-puzzle/issues/6#issuecomment-842580474_

**Comment by Naddiseo** [comment]
For the QR code, it's most like a compression artifact. shifting from png -> jpg will create similar patterns.

**Comment by nukucux** [comment]
![image](https://user-images.githubusercontent.com/82031466/119443521-f24d0080-bd53-11eb-8063-39a752ad2eb6.png)
using sweep level analysis, I found 3 colours in the image

**Comment by nukucux** [comment]
![final gsmg](https://user-images.githubusercontent.com/82031466/119490111-a8c9d900-bd86-11eb-9e58-e970bdaf39f8.png)
on sweep analysis

**Comment by slonyarakmw** [comment]
> There are 24 steps, so. 24x8=192
> 
> Tiger hash?

No.. It's don't need to encrypt. It's just hint to conter-clock wise way. Just telling us where we need to go by encrypting  this squares (by 1 and 0)

**Comment by nukucux** [comment]
> Ok. I thought about this squares a lot. (blue and yellow), and finally, i solved this thing (just i think so). Because this is one logical reason, why they on their places..     If we start to count from left top square counterclock-wise, we'll get 1 square every 8 step.  If  it ends on black - it's blue. If white - yellow... 
> 
> ![Another One](https://user-images.githubusercontent.com/40121870/120228278-7bcc6900-c253-11eb-8ae8-b1ab81801ee8.jpg)
> 
> 
> That's why i'm actually think, that this "markers" are just a "Hint" to the right way.
> F#ck....

There are 24 steps, so. 24x8=192

Tiger hash?

**Comment by slonyarakmw** [comment]
> There are 24 steps, so. 24x8=192
> 
> Tiger hash?

No.. It's don't need to encrypt. It's just hint to conter-clock wise way. Just telling us where we need to go to encrypting  this black and white squares (by 1 and 0)

**Comment by heellrzz** [comment]
I suggest trying to create a path apart from the rabbit that can reach the yellow pixel in the upper left corner
